_shared/project/tox.ini

```diff
@@ -104,7 +104,6 @@ commands =
     lint: pylint --rcfile=tests/pyproject.toml tests
     lint: pydocstyle {{ cookiecutter.package_name }} tests bin
     lint: pycodestyle {{ cookiecutter.package_name }} tests bin
-    dockercompose: docker-compose {posargs}
```

Comment:
No more need for any `docker-compose` stuff at all in `tox.ini` now that `Makefile` just runs the `docker compose` command that ships with Docker Desktop instead. This is also the reason why `requirements/dockercompose.in` was removed.